frontend/cypress/integration/common/overview.ts

```diff
@@ -0,0 +1,135 @@
+import { And, Before, Given, Then } from "cypress-cucumber-preprocessor/steps";
+
+const url = "/console";
+
+Before(() => {
+    // Focing to not stop cypress on unexpected errors not related to the tests.
+    // There are some random failures due timeouts/loadtime/framework that throws some error in the browser.
+    // After reviewing the tests failures, those are unrelated to the app, so,
+    // it needs this event to not fail the CI action due some "slow" action or similar.
+    // This is something to review in future iterations when tests are solid, but I haven't found a better way to
+    // solve this issue.
```

Comment:
From what I saw, the compiled final .js had some pull errors.
So, I've spent some time on it and it was unrelated to the test, but it was adding noise with pipeline failures.
It will require some investigation but with this hack, the tests passed well.

This should be iterated, but when coding "real" tests with some complexities we'll have to introduce some code like this.

Reply:
If there are specific errors that we expect to be thrown we should catch those and ignore them scoping the error handling to the specific test. I'm worried that ignoring all unhandled exceptions will hide real issues that we could've caught as part of the tests. Have you tried removing this since setting the `refresh=0`? I'm wondering if the intermittent errors you were seeing were caused by the unhandled promise rejections.

Reply:
Totally agree. Nobody wants a flaky pipeline. Can we just catch unhandled promise rejections for now and see if that solves the intermittent failure issue?
```
Cypress.on('uncaught:exception', (err, runnable, promise) => {
  // when the exception originated from an unhandled promise
  // rejection, the promise is provided as a third argument
  // you can turn off failing the test in this case
  if (promise) {
    return false
  }
  // we still want to ensure there are no other unexpected
  // errors, so we let them fail the test
})
```

Reply:
What are these "pull errors"? Do we have some logs, or steps to reproduce? 
I don't want to discard all uncaught exceptions.

Reply:
These errors are similar to these ones [1]

![image](https://user-images.githubusercontent.com/1662329/161706809-6d6d05a8-3980-4877-a8c8-e3547d7dbeee.png)

And it seems that are related to the situation that when we click in that button, there is a request to the server, and randomly the cypress framework detects some promise not completed.

That button doesn't have a problem, it's a matter of how the framework is setup and we will find more scenarios like this, so we need to introduce some tradeoffs, as we can't have an unreliable pipeline due to some random errors (timeouts, infrastructure, etc). 

I'm testing Nick's proposal to see if this mitigates the problem.

[1] https://github.com/kiali/kiali/runs/5815529616?check_suite_focus=true

Reply:
@nrfox your comment seems to work, I've updated the PR.

Reply:
@ScriptingShrimp I did a pass of the wording in the feature with better use of the "aliases". 
Thanks for the suggestions, let me know how it looks.

I'm less worried about the "content" as we will add new tests and modify the existing ones, but I'd like that at least this work may help others to create more tests.